src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -55,5 +55,13 @@ public static class KnownQueryParameterNames
         public const string Id = "_id";
 
         public const string Type = "_type";
+
+        /// <summary>
+        /// The anonymization configuration
+        /// </summary>
+        /// <remarks>The anonymization configuration location and addition information. </remarks>
+        public const string AnonymizationConfigurationLocation = "_anonymization_config";
```

Comment:
should these be camel case? e.g. `_lastUpdated`

Reply:
Yes, updated to follow the same convention. Thanks